src/Tools/BuildValidator/LocalReferenceResolver.cs

```diff
@@ -44,12 +59,32 @@ public static DirectoryInfo GetNugetCacheDirectory()
             return new DirectoryInfo(nugetPackageDirectory);
         }
 
+        public static DirectoryInfo[] GetRefAssembliesDirectories()
+        {
+            // TODO
```

Comment:
```suggestion
            // TODO: Don't hardcode the paths here. 
```

Can we make sure this is tracked in the issue?